src/Event/PostMappingBuildEvent.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/*
+ * This file is part of the FOSElasticaBundle package.
+ *
+ * (c) FriendsOfSymfony <https://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\ElasticaBundle\Event;
+
+use FOS\ElasticaBundle\Configuration\IndexConfigInterface;
+use Symfony\Contracts\EventDispatcher\Event;
+
+class PostMappingBuildEvent extends Event
```

Comment:
You may make it final.

Reply:
Ok will do. But that makes it different than other events?

Reply:
https://github.com/FriendsOfSymfony/FOSElasticaBundle/pull/1734/commits/ffd887dd38cc0998d1a0fa7b85d3c45748c640ea

Reply:
@XWB we could apply this to other event before publising stable version

Reply:
Good idea!